doc/tutorials/acrn_configuration_tool.rst

```diff
@@ -136,17 +139,26 @@ Additional scenario XML elements:
 ``name`` under parent of ``os_config``:
   Specify the OS name of VM; currently, it is not referenced by the hypervisor code.
```

Comment:
Is that OS name of VM name?

Reply:
It's the OS name. For example, "Ubuntu 18.04", "ClearLinux xxxxx". Currently it is not used, but it might be used in the future if hypervisor have specific operation for specific OS.

Reply:
Thanks for the clarification!